runtime/moonbase/src/lib.rs

```diff
@@ -638,13 +662,15 @@ impl pallet_identity::Config for Runtime {
 	type MaxSuffixLength = MaxSuffixLength;
 	type MaxUsernameLength = MaxUsernameLength;
 	type WeightInfo = moonbase_weights::pallet_identity::WeightInfo<Runtime>;
+	type UsernameDeposit = ();
+	type UsernameGracePeriod = ();
```

Comment:
Maybe 30 days?
```suggestion
	type UsernameGracePeriod = ConstU32<{ 30 * DAYS }>;
```

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/eff30bd3ef8cecd3bbed397e753d39d07f58f77a